databaseconnect.py

```diff
@@ -169,52 +171,55 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")    # noqa: E501
+    cur.execute("UPDATE statement_table SET sentence = '{}' WHERE id = '{}'".format(H, last_id))  # noqa: E501
     db.commit()
     B = "Thank you! I have learnt this."
     return B, 0
 
 
+@logger_config.logger
 def clear_table(table_name):
     db = connection_to_database()
     cur = db.cursor()
 
     if table_name in ("question_table", "statement_table"):
         tables_to_be_cleaned = ("question_table", "statement_table")
-        print("The following tables will be cleaned:\n")
+        cur.execute("The following tables will be cleaned:\n")
```

Comment:
I don't think this change is valid. AFAIK, `cur.execute` will take SQL queries, not the statements to be printed out on the screen.